src/components/forms/LoginForm/LoginForm.tsx

```diff
@@ -36,14 +48,28 @@ export const LoginForm = () => {
     const {
         register,
         handleSubmit,
-        reset,
         formState: { errors },
-    } = useForm<LoginData>({
+    } = useForm<LogInFormValues>({
         resolver: yupResolver(logInSchema),
         defaultValues,
     })
 
-    const { onSubmit } = useFormSubmit<LoginData>({ reset, method: 'post' })
+    const onSubmit = async (data: LogInFormValues): Promise<any> => {
+        try {
+            const response = await fetch(`${apiUrl}${ENDPOINTS.signIn}`, {
+                method: 'POST',
+                headers: { 'Content-Type': 'application/json' },
+                body: JSON.stringify(data),
+            })
+            if (response.ok) {
+                setIsWrongAccount(false)
+            } else {
+                setIsWrongAccount(true)
+            }
```

Comment:
czy tutaj mozemy krocej
setIsWrongAccount(response(response.ok ? false : true);

Reply:
a i co do walidacji moim zdaniem to fajnie ze tu jest, bo uzytkownik ma info co robi źle, kiedy nie moze sie zalogować -wiec dbamu o User Experience :)

Reply:
myślę, że mogę to skrócić spokojnie